streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/StateStoreMetrics.java

```diff
@@ -443,6 +447,25 @@ public static Sensor suppressionBufferSizeSensor(final String threadId,
         );
     }
 
+    public static Sensor e2ELatencySensor(final String threadId,
+                                          final String taskId,
+                                          final String storeType,
+                                          final String storeName,
+                                          final StreamsMetricsImpl streamsMetrics) {
+        final Sensor sensor = streamsMetrics.storeLevelSensor(threadId, taskId, storeName, RECORD_E2E_LATENCY, RecordingLevel.TRACE);
+        final Map<String, String> tagMap = streamsMetrics.storeLevelTagMap(threadId, taskId, storeType, storeName);
+        addAvgAndMinAndMaxToSensor(
+            sensor,
+            STATE_STORE_LEVEL_GROUP,
```

Comment:
You need to use the `stateStoreLevelGroup()` here instead of `STATE_STORE_LEVEL_GROUP` because the group name depends on the version and the store type.

Reply:
I just realized that we should not put new metrics into old groups. Your code is fine. Do not use `stateStoreLevelGroup()`! Sorry for the confusion.